cfgov/jinja2/v1/_includes/organisms/half-width-link-blob-group.html

```diff
@@ -0,0 +1,5 @@
+<div class="o-link-blob-group">
+    {% for block in value.link_blobs %}
```

Comment:
In #1171 it's just an `h2`. I actually have a commit coming that will add headers to the groups.